packages/tinacms/src/hooks/use-graphql-forms.tsx

```diff
@@ -287,7 +293,7 @@ export function useGraphqlForms<T extends object>({
         console.error(e)
         setIsLoading(false)
       })
-  }, [queryString, JSON.stringify(variables)])
+  }, [queryString, JSON.stringify(variables), refreshVar])
```

Comment:
@logan-anderson when `cms:forms:refresh` fires will the wipe out form values that have been edited?